test/test_suggest.py

```diff
@@ -1,11 +1,37 @@
-'''
-Example test file
+'''file contains modules to test modules in suggestion.py'''
 
-New test files should be created for each module
-'''
-# pylint: disable=missing-function-docstring
+#import pytest
+
+import pandas as pd
 from what_to_watch.suggest import suggestion
 
+DF_DATA = {'item':  ['Kazam', 'big chicken shaq'],
+           'rating': ['10', '10'],
+           'type': ['movie', 'tv_show']
+           }
+
+DF = pd.DataFrame(DF_DATA, columns=['item', 'rating', 'type'])
+ARGS_NULL = ['movies']
+ARGS_MOVIE = ['movie']
+ARGS_TV_SHOW = ['tv_show']
+
+def test_suggestion_null():
+    '''function to test suggest_movie module
+    should return null
+    as args value is movies, not movie or tv_show'''
+
+    assert (suggestion.suggest_movie(ARGS_NULL, DF)) == "space jam"
```

Comment:
We should problem raise an Error in this case right? Maybe a `ValueError` instead of silently failing and returning "space jam"?